docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -1,51 +1,21 @@
 # Getting Started - Advocated Pattern Owner
```

Comment:
Based on our discussion earlier this week, shouldn't we remove any designated persona reference as our target audience is just developers?